docs/contributing/how-to-release.md

```diff
@@ -171,20 +172,32 @@ WINDOWS71SDK
 
 If either of these are blank they are set to the default install location.
 
+For Visual Studio 2017 and 2019 the VSxxxxCOMNTOOLS environment variable is not
+set by the installer. Each of these studio version supports multiple toolsets.
```

Comment:
I would change "installer. Each of these studio version supports" to "as these two versions support".